modules/gapi/test/gapi_array_tests.cpp

```diff
@@ -264,7 +264,7 @@ TEST(GArray_VectorRef, Kind)
     EXPECT_EQ(cv::detail::OpaqueKind::CV_SIZE, v7.getKind());
 
     cv::detail::VectorRef v8(std::vector<std::string>{});
-    EXPECT_EQ(cv::detail::OpaqueKind::CV_UNKNOWN, v8.getKind());
+    EXPECT_EQ(cv::detail::OpaqueKind::CV_STRING, v8.getKind());
```

Comment:
We still need to test the types we don't know result in `CV_UNKNOWN`. The easiest way to guarantee there is a type G-API doesn't now is to define this type right in this test.

Reply:
Right, added a check on unknown type/kind